api/types.go

```diff
@@ -111,6 +111,16 @@ type Message struct {
 	Images  []ImageData `json:"images,omitempty"`
 }
 
+type CompletionProbability struct {
+	Content string            `json:"content"`
+	Probs   []CompletionProbs `json:"probs"`
+}
+
+type CompletionProbs struct {
```

Comment:
You'd better add comments for struct, refer some struct comments in this file(`api/types.go`). e.g:
```go
// StatusError is an error with and HTTP status code.
type StatusError struct {
	StatusCode   int
	Status       string
	ErrorMessage string `json:"error"`
}
```
`// StatusError is an error with and HTTP status code.` is comment for struct `StatusError`

Reply:
Fixed.
Personal opinion for types with like <5 elements: That's probably a bit over-commenting. But i see the argument for consistency here.